boards/adi/max32657evkit/doc/index.rst

```diff
@@ -0,0 +1,385 @@
+featuring an ARM Cortex-M33 architecture that provides Trustzone technology
+which allow define secure and non-secure application.
+
+The BOARD options are summarized below:
+
++-------------------------------+-------------------------------------------+
+| BOARD                         | Description                               |
++===============================+===========================================+
+| max32657evkit/max32657        | For building Trust Zone Disabled firmware |
++-------------------------------+-------------------------------------------+
+
+
+BOARD: max32657evkit/max32657
+=============================
+
+Build the zephyr app for ``max32657evkit/max32657`` board will generate secure firmware
+for zephyr. In this configuration 960KB of flash is used to store the code and 64KB
+is used for storage section. In this mode tf-m is off and secure mode flag is on
+``:kconfig:option:CONFIG_TRUSTED_EXECUTION_SECURE=y`` and
+``:kconfig:option:CONFIG_BUILD_WITH_TFM=n``
+
++----------+------------------+---------------------------------+
+| Name     | Address[Size]    | Comment                         |
++==========+==================+=================================+
+| slot0    | 0x1000000[960k]  | Secure zephyr image             |
++----------+------------------+---------------------------------+
+| storage  | 0x10f0000[64k]   | File system, persistent storage |
++----------+------------------+---------------------------------+
+
+Here are the instructions to build zephyr with a secure configuration,
+using :zephyr:code-sample:`blinky` sample:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/blinky/
+   :board: max32657evkit/max32657
+   :goals: build
+
+
+Programming and Debugging
+*************************
+
+.. zephyr:board-supported-runners::
+
+Flashing
+========
+
+Here is an example for the :zephyr:code-sample:`hello_world` application. This example uses the
+:ref:`jlink-debug-host-tools` as default.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: max32657evkit/max32657
+   :goals: flash
+
+Open a serial terminal, reset the board (press the RESET button), and you should
+see the following message in the terminal:
+
+.. code-block:: console
+
+   ***** Booting Zephyr OS build v4.1.0 *****
+   Hello World! max32657evkit/max32657
+
+
+Debugging
+=========
+
+Here is an example for the :zephyr:code-sample:`hello_world` application. This example uses the
+:ref:`jlink-debug-host-tools` as default.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: max32657evkit/max32657
+   :goals: debug
+
+Open a serial terminal, step through the application in your debugger, and you
+should see the following message in the terminal:
+
+.. code-block:: console
+
+   ***** Booting Zephyr OS build v4.1.0 *****
+   Hello World! max32657evkit/max32657
+
+
+References
+**********
+
+.. _1:
+   https://developer.arm.com/documentation/100935/0100/The-TrustZone-hardware-architecture-
```

Comment:
What's your intent with this?
It needs to be actually used in the document to show up to the reader

Reply:
Removed from this PR will be valuable for https://github.com/zephyrproject-rtos/zephyr/pull/88706
Thanks.